src/simulation/Element.h

```diff
@@ -41,6 +41,7 @@ class Element
 	int Weight;
 	unsigned char HeatConduct;
 	unsigned int Properties;
+	uint16_t HudProperties;
```

Comment:
Either make it an `uint32_t` or stick to the `unsigned int` pattern. No need to be smart about integer sizes.